docs/source/etcd-data-model.rst

```diff
@@ -0,0 +1,243 @@
+Calico etcd Data Model
+======================
+
+The current version of Calico is built around `etcd`_. etcd is a distributed,
+consistent key value store for shared configuration and service discovery with
+a focus on being simple, secure, fast, and reliable.
+
+In Calico, etcd is used as the data store and communication mechanism for all
+the Calico components. This data store contains all the information the various
+Calico components to set up the Calico network.
+
+This document discusses the way Calico stores its data in etcd. This data store
+and structure acts as Calico's primary external and internal API, granting
+developers exceptional control over what Calico does. This document does not
+discuss the way this data gets turned into actual network topologies by
+individual components: for more on that, see :doc:`etcd-architecture`.
```

Comment:
I'm not sure `etcd-architecture` discusses that either....  Perhaps a better wording for what you mean would be "This document does not describe the components that read and write this data so as to provide the connectivity that endpoints in a Calico network want: for more on that ...."